frontend/src/console/components/CoursesManage/CoursesManage.tsx

```diff
@@ -0,0 +1,113 @@
+import * as React from 'react';
+import { InstancesModel } from 'console/models';
+import {
+  ConsolePage,
+  ConsolePageCustomizationContainer
+} from 'console/components';
+import { CONTACT_US_LINK, ROUTES } from 'global/constants';
+import { WrappedMessage } from 'utils/intl';
+import { Container, Col, Button, Row } from 'react-bootstrap';
+import { RootState } from 'global/state';
+import { connect } from 'react-redux';
+import messages from './displayMessages';
+import './styles.scss';
+
+interface State {
+  title: string;
+  subtitle: string;
+  // extra state to manage the empty title and subtitle and rendering
+  renderBool: boolean;
+}
+
+interface ActionProps {
+  clearErrorMessage: Function;
+  updateThemeFieldValue: Function;
+  updateImages: Function;
+}
+
+interface StateProps extends InstancesModel {}
+
+interface Props extends StateProps, ActionProps {}
+
+export class CoursesManageComponent extends React.PureComponent<Props, State> {
+  public render() {
+    let instanceLink;
+    if (this.props.activeInstance.data) {
+      instanceLink = this.props.activeInstance.data.studioUrl;
+    } else {
+      instanceLink = ROUTES.Error.UNKNOWN_ERROR;
+    }
+
+    return (
+      <ConsolePage contentLoading={this.props.loading}>
+        <div className="courses-page">
+          <Container fluid className="container">
```

Comment:
Remove the surrounding `<Container fluid className="container">`. You already have a container element:
```suggestion
```